src/Plugin/Field/FieldFormatter/InstagramEmbedFormatter.php

```diff
@@ -34,10 +38,16 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
     foreach ($items as $delta => $item) {
       $matches = [];
       foreach (Instagram::$validationRegexp as $pattern => $key) {
-        if (preg_match($pattern, $item->value, $matches)) {
+        if (preg_match($pattern, $this->getEmbedCode($item), $item_matches)) {
+          $matches[] = $item_matches;
           break;
         }
       }
+
+      if (!empty($matches)) {
+        $matches = reset($matches);
```

Comment:
hmm.. I was wondering why is there this reset, but if I drop that check then in the test is missing the formatter which is what we want to assert. Why is it so?